packages/theme-chalk/src/dialog.scss

```diff
@@ -12,6 +12,8 @@
   box-shadow: $--dialog-box-shadow;
   box-sizing: border-box;
   width: 50%;
+  max-height: 70%;
```

Comment:
Why magical 70%?

Reply:
I just causally to set the value.

Reply:
I think we need to use `flex` to layout the UI, set the content to `flex:1`, also set `overflow` to `scroll`, since `70%` isn't a legit number to me.

Reply:
In your way, the height of dialog will still overflow its father div. It seems that the value of margin can't let the height of dialog compress.
![image](https://user-images.githubusercontent.com/57524062/102170376-2da6f180-3ecf-11eb-9ab6-75b98bf8466f.png)

Reply:
Maybe we can try something like making the `el-overlay` to be able to be scrollable.
Like when the content gets very long, the whole background can be scrolled.
Maybe this is a better way to do this, I suppose. BTW, the v2 version acts like this.

Reply:
Although the idea is that dialog shouldn't be putting too much stuff inside, but we cannot control how the developers/PMs behavior, so we can try this way. 
The idea is to find a better, elegant way to solve the issue, I want to hear your voice here.

Reply:
Well, I use the way of v2. I think it may be a good way because we can't control the height of dialog.